TrackCat/templatetags/navigation.py

```diff
@@ -0,0 +1,27 @@
+from django import template
+from django.core import urlresolvers
+ 
```

Comment:
you have single spaces in these "empty" lines... might as well remove them :wink: